src/MLHeader/MLHeader.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { PageHeader } from 'antd'
+import classNames from 'classnames'
+
+const MLHeader = (props) => {
+  const avatarProps = { size: 48, ...props.avatar }
+  return (
+    <PageHeader
```

Comment:
I'm torn on if this should use PageHeader or MLPageHeader.

Reply:
Good question. Using the MarkLogic component is more consistent with what I've been trying to do elsewhere. I added a comment warning that changes there will affect this component. It will show up in DocsPage if we end up using that.